src/azure/cli/commands/parameters.py

```diff
@@ -65,6 +65,9 @@ def completer(prefix, action, parsed_args, **kwargs): # pylint: disable=unused-a
         return list(enum_type.__dict__[ENUM_MAPPING_KEY].keys())
     return completer
 
+def get_enum_choices(enum_type):
+    return [x.value for x in enum_type]
```

Comment:
this returns the array for a choices param, the other method returns a completer that uses a property I don't need for the choices param

---
In reply to: [74151895](https://github.com/Azure/azure-cli/pull/668#discussion_r74151895) [](ancestors = 74151895)